MapboxStatic.xcodeproj/project.pbxproj

```diff
@@ -1071,6 +1125,10 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 7;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(inherited)",
+					"$(PROJECT_DIR)/Carthage/Build/iOS",
```

Comment:
This works but I think the recommended way is https://github.com/Carthage/Carthage#if-youre-building-for-ios-tvos-or-watchos